### PR TITLE
hotfix: `openChatWindow` function

### LINKED ISSUE #3287: Error opening chat window - window.Store.Cmd.openChatAt("")
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

getting the following error while trying to open any chat 

```
Uncaught TypeError: Cannot read properties of undefined (reading 'msgs')
    at f.openChatBottom (YJmClXJHQ5E.js?_nc_x=Ij3Wp8lg5Kz:1440:10176)
    at f.openChatAt (YJmClXJHQ5E.js?_nc_x=Ij3Wp8lg5Kz:1440:7819)
    at <anonymous>:1:32
```

How can I fix it? 
PS: This was working till yesterday

### Expected behavior

Expected behavior - Chat window should have been opened

### Steps to Reproduce the Bug or Issue

Create a whatsapp web client and call 
```
client.interface.openChatWindow("your-contact@c.us")
```

### Relevant Code

_No response_

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Windows
Whatsapp web version - 2.3000.1016746436

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by alechkos**
# PR Details

The PR fixes an issue where the `InterfaceController.openChatWindow` function throws an error:
```js
Uncaught TypeError: Cannot read properties of undefined (reading 'msgs')
```

Thank's to @BenyFilho for the [bug resolution](https://github.com/pedroslopez/whatsapp-web.js/issues/3287#issuecomment-2371241893).

## Related Issue

closes #3287

## How Has This Been Tested

```js
// client initialization...
await client.interface.openChatWindow('XXXXXXXXXX@c.us');
```

## Types of changes

- [ ] Dependency change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Review by shirser121 — APPROVED** [review]
LGTM

**Review by tuyuribr — APPROVED** [review]

File: src/util/InterfaceController.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ class InterfaceController {
      */
     async openChatWindow(chatId) {
         await this.pupPage.evaluate(async chatId => {
-            let chatWid = window.Store.WidFactory.createWid(chatId);
-            let chat = await window.Store.Chat.find(chatWid);
-            await window.Store.Cmd.openChatAt(chat);
+            const chatWid = window.Store.WidFactory.createWid(chatId);
+            const chat = window.Store.Chat.get(chat) || await window.Store.Chat.find(chatWid);
```
Comment (StiflerPasha) [inline src/util/InterfaceController.js:19]: In `window.Store.Chat.get(chat)`, `chat `was used before it was defined

Comment (alechkos) [reply src/util/InterfaceController.js:19]: Thank you

Comment (Milton-scz) [reply src/util/InterfaceController.js:19]: no working

**Pull request merged**